ldap/servers/slapd/pw.c

```diff
@@ -2356,6 +2356,18 @@ new_passwdPolicy(Slapi_PBlock *pb, const char *dn)
                     if ((sval = attr_get_present_values(attr))) {
                         pwdpolicy->pw_dict_path = (char *)slapi_value_get_string(*sval);
                     }
+                } else if (!strcasecmp(attr_name, CONFIG_PW_TPR_MAXUSE)) {
+                    if ((sval = attr_get_present_values(attr))) {
+                        pwdpolicy->pw_tpr_maxuse = (char *)slapi_value_get_int(*sval);
```

Comment:
Is there a reason you cast int to 'char *'? Seems a bit odd...

Reply:
Good catch.. it was a bad copy/paste :)
Updated the patch